rust-go-security-engineer.md

```diff
@@ -0,0 +1,57 @@
+# Wanted: Rust/Go Security Engineer
+
+Are you passionate about securing decentralised systems and exploring the intricacies of Rust and/or Go programming languages? Sigma Prime is seeking a talented Rust/Go Security Engineer to join our team, focusing on conducting Blockchain security assessments for projects built in Rust and Go.
+
+## About Us
+
+At **Sigma Prime**, we are dedicated to building a secure and decentralised world. As a leading blockchain security and research firm, Since 2015, Ethereum has been a focal point of our work, and we've collaborated with leading companies in the blockchain space, offering security reviews, design consultancy, and niche development services.
```

Comment:
```suggestion
**Sigma Prime** is a leading blockchain and security research firm,  dedicated to building a secure and decentralised world.  Since we started in 2015, Ethereum has been a focal point of our work, and we've collaborated with some of the leading companies in the blockchain space, offering security reviews, design consultancy, and niche development services in a range of technical capacities.
```